docs/aws/audit/cloudwatchmonitoring/rules/webacl_rules_in_count_mode.mdx

```diff
@@ -0,0 +1,169 @@
+</Accordion>
+
+<Accordion title='Using CLI'>
+WebACL rules should not be in count mode because this mode only counts the web requests that match the properties of a rule, but does not block or allow them. Here are the steps to remediate this misconfiguration using AWS CLI:
+
+Step 1: Install AWS CLI
+Firstly, you need to install AWS CLI on your system. You can download it from the official AWS website. 
+
+Step 2: Configure AWS CLI
+Once installed, you need to configure AWS CLI with your AWS account details. You can do this by running the following command in your terminal:
+
+```
+aws configure
+```
+
+You'll be prompted to enter your AWS Access Key ID, Secret Access Key, default region name, and default output format. Enter these details correctly.
+
+Step 3: List WebACLs
+Next, you need to list all the WebACLs in your account. You can use the following command:
+
+```
+aws wafv2 list-web-acls --scope REGIONAL
+```
+
+This will list all the WebACLs in your account.
+
+Step 4: Identify WebACLs in Count Mode
+Go through the list of WebACLs and identify the ones that are in count mode. You can do this by checking the "DefaultAction" field of each WebACL. If it is set to "COUNT", the WebACL is in count mode.
+
+Step 5: Change DefaultAction to Block or Allow
+Once you've identified the WebACLs in count mode, you need to change their DefaultAction to either "BLOCK" or "ALLOW". You can use the following command:
+
+```
+aws wafv2 update-web-acl --name <name> --scope REGIONAL --id <id> --default-action Allow={} --lock-token <lock-token>
+```
+
+Replace <name> with the name of the WebACL, <id> with the ID of the WebACL, and <lock-token> with the lock token of the WebACL. This command will change the DefaultAction of the WebACL to "ALLOW". If you want to change it to "BLOCK", replace Allow={} with Block={}.
+
+Step 6: Verify the Changes
+Finally, verify the changes by listing the WebACLs again and checking their DefaultAction. If it is set to "BLOCK" or "ALLOW", the remediation was successful.
+
+Remember to replace all the placeholders with actual values. Also, ensure that you have the necessary permissions to perform these actions.
+</Accordion>
+
+<Accordion title='Using Python'>
+The WebACL rules in count mode essentially means that the rule is only counting the matches for a rule without any action. This configuration can be useful for testing, but in a production environment, it's recommended to change it to block or allow mode.
+
+Here's how to remediate this misconfiguration using AWS SDK for Python (Boto3):
+
+1. First, install Boto3 if it's not already installed. You can do this by running the following command in your terminal:
+   ```
+   pip install boto3
+   ```
+
+2. Import the necessary modules and set up your AWS credentials:
+
+   ```python
+   import boto3
+   from botocore.exceptions import NoCredentialsError
+
+   ACCESS_KEY = 'YOUR_ACCESS_KEY'
+   SECRET_KEY = 'YOUR_SECRET_KEY'
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code/findings/d803ac66c50fdc16/PR/146/docs/puru-automation/latest/secrets/show" target='_blank'>Click here for more details</a></b>